src/lib/chain/offer.ts

```diff
@@ -257,68 +276,153 @@ export async function updateOffers(flags: OffersArgs) {
     );
   }
 
-  for (const { minPricePerWorkerEpochBigInt, id } of offersToUpdate) {
-    const offerInfo = await market.getOffer(id);
-    const dealExplorerClient = await getDealExplorerClient();
-    const offerExplorerInfo = await dealExplorerClient.getOffer(id);
-    // TODO: USE IT
-    void offerExplorerInfo;
+  for (const {
+    minPricePerWorkerEpochBigInt,
+    offerId,
+    effectors,
+    offerInfo: { offerInfo, offerExplorerInfo } = {
+      offerInfo: undefined,
+      offerExplorerInfo: undefined,
+    },
+  } of offersToUpdate) {
+    if (offerInfo === undefined || offerExplorerInfo === undefined) {
+      commandObj.warn(
+        `Can't find offer info for offer with id ${offerId} in chain. Please check if the offer exists in chain and try again. If the offer doesn't exist you can create it using '${CLI_NAME} provider offer-create' command`,
```

Comment:
```suggestion
        `Can't find offer info for offer with id ${offerId} in chain. Please check whether the offer exists in chain and try again. If the offer doesn't exist you can create it using '${CLI_NAME} provider offer-create' command`,
```

Reply:
I think both variants are acceptable. I prefer "if" cause it's shorter, less to read, less formal. I don't like to be unnecessarily formal in communications with developers

Reply:
I think that they convey different meaning despite they could be used interchangeably sometimes.
`Whether` is used when there are 2 alternatives and you basically underlining that. 
In you case, the alternatives - offer does exist or doesn't.
Then you use `if` to formalize condition (if **A** happends, you can do **B**). 
I think that this distinction is kinda beneficial for developer, because more clearly passes the text meaning instead of putting `if` everywhere.

Reply:
I really think this can be either way, but if you really like it better like that I don't mind